src/components/disclaimer/disclaimer.js

```diff
@@ -8,16 +8,17 @@ import {
 })
 export class DisclaimerComponent {
   @Inject() $cookies;
+  @Inject() disclaimerStore;
 
   activate() {
     if (this.$cookies.get('disclaimer-accepted') !== 'true') {
-      this.showDisclaimer = true;
+      this.disclaimerStore.activateDisclaimer();
```

Comment:
You should never call functions directly on the store to manipulate data. Please create an disclaimer-actions.js in which you  have an activate and hidedisclaimer function. And listen in the store to those.